acceptance/analyzer_test.go

```diff
@@ -868,6 +874,27 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 						})
 					})
 				})
+
+				when("do have read access to registry", func() {
+					it("throw read error", func() {
+						h.SkipIf(t, api.MustParse(platformAPI).Compare(api.MustParse("0.7")) < 0, "Platform API < 0.7 does not use tag flag")
```

Comment:
Why do we have this skip? Did this come from copy/paste?

Reply:
Yeah, it's a copy pace issue 😞

Reply:
I removed it in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/04d939f1fc0b51330b0919208870efe1317f1775)